test/e2e-go/e2e_go_olm_test.go

```diff
@@ -63,6 +63,17 @@ var _ = Describe("Integrating Go Projects with OLM", func() {
 				Expect(err).NotTo(HaveOccurred())
 			}
 
+			By("running the operator bundle using `run bundle` command")
+			runBundleCmd := exec.Command(tc.BinaryName, "run", "bundle", bundleImage, "--namespace", tc.Kubectl.Namespace)
+			_, err := tc.Run(runBundleCmd)
+			Expect(err).NotTo(HaveOccurred())
+
+			By("destroying the Operator deployed with the 'run' subcommand")
+			cleanupPkgManCmd := exec.Command(tc.BinaryName, "cleanup", projectName,
+				"--timeout", "4m")
+			_, err = tc.Run(cleanupPkgManCmd)
+			Expect(err).NotTo(HaveOccurred())
+
```

Comment:
👍

Reply:
It shows just missing call the run bundle with Ansible/Helm as well to esnure that it will work there as well. So, it means copy and past what was done here for  `test/e2e-helm/e2e_helm_olm_test.go` and `test/e2e-ansible/e2e_ansible_olm_test.go`